packages/core/src/molecule/codec.ts

```diff
@@ -18,13 +18,19 @@ import {
 
 export type CodecLike<Encodable, Decoded = Encodable> = {
   readonly encode: (encodable: Encodable) => Bytes;
-  readonly decode: (decodable: BytesLike) => Decoded;
+  readonly decode: (
+    decodable: BytesLike,
+    config?: { isExtraFieldIgnored?: boolean },
```

Comment:
Use term `compatible` which is already used in Rust(https://github.com/search?q=repo%3Anervosnetwork%2Fmolecule%20compatible&type=code) instead of `isExtraFieldIgnored`

Reply:
> I thought `isCompatible` should be a good enough name here. But after consideration, I suspect that `compatible` is an accurate and descriptive word here.

> If we say `compatible` or `isCompatible`, it sounds more like the `decodable` is `compatible`, while it's not. It's more like we are telling "use the compatible mode", so `isCompatibleMode`  might be more accurate, but still not descriptive enough.

> As a result, I suggest describing exactly what we're doing, like `shouldAcceptExtraField`/`isExtraFieldIgnored`. And add a comment here to tell this is the same as `compatible` in Rust.